src/root.component.tsx

```diff
@@ -1,10 +1,19 @@
 import React from "react";
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter, Route } from "react-router-dom";
 import Topnav from "./topnav/topnav.component";
 
 export default function Root(props: RootProps) {
   return (
     <BrowserRouter basename="/openmrs/spa">
+      <Route
```

Comment:
I think that handling logging out should be done inside of openmrs-esm-login instead of openmrs-esm-navbar. So this route would be better added to https://github.com/openmrs/openmrs-esm-login than here.

For now, though, we can just delete this route entirely until we are ready to implement it in openmrs-esm-login